src/main/java/edu/harvard/h2ms/service/report/AbstractReportWorkerComplianceWarning.java

```diff
@@ -37,15 +37,62 @@
 
   long REPORTINGINTERVAL = NotificationFrequency.WEEKLY.seconds;
 
+  public final String COMPLIANCETHRESHOLD = "complianceThreshold";
```

Comment:
Words in variable name should be separated with an underscore, aka `COMPLIANCE_THRESHOLD`